core/domain/app_feedback_report_domain_test.py

```diff
@@ -161,7 +161,7 @@ def test_to_dict_android_report(self) -> None:
             'scrubbed_by': None,
             'user_supplied_feedback': {
                 'report_type': REPORT_TYPE_SUGGESTION.name,
-                'category': CATEGORY_SUGGESTION_OTHER.name,
+                'category': CATEGORY_SUGGESTION_OTHER.value,
```

Comment:
Why is the one above ``name`` but this one ``value``? They both seem migrated to the new style, but I'm not sure when ``value`` vs. ``name`` should be used.

Reply:
Thanks Ben, you're right, I hadn't noticed that and now I'm a bit confused too. The PR which changed the style of the `ReportType` enum is #15296, and the diff shows that in _some_ places, the enum was indeed changed to use the `.value` property, but as you pointed out here, in some places it was left unchanged.

As for the difference between `name` and `value`, I think it's just the difference between the key and the value of the enum. Earlier both were in lowercase so both `.name` and `.value` resulted in the same string, but now the keys are changed to UPPERCASE, so since `.name` refers to the key and `.value` refers to the value, both properties are now resulting in different strings.

Reply:
I see the main benefit of using ``value`` over ``name`` to be that ``value`` can be kept with the same contents while being able to rename constants. This is really nice for long-term maintainability and future migrations that these structures may undergo.